healthyregions/chicago-environment-explorer
# fix: cleanup/address build warnings

## Problem
There are quite a few warnings that appear when running `yarn build`

Fixes #189

## Approach
* fix: all build warnings
    * ~~NOTE: there is one warning that we have addressed, but the warning has not gone away: `@babel/plugin-proposal-private-property-in-object`~~

## How to Test
1. Checkout this repo locally
2. Run `yarn install` to update, install, and/or prune dependencies
3. Run `yarn build`
    * You should see that the build warnings no longer appear
4. Navigate to the PR preview, to verify that nothing broke as a result of cleaning up these warnings
